config/interceptor/interceptor.yaml

```diff
@@ -0,0 +1,86 @@
+---
+apiVersion: apps/v1
+kind: Deployment
+metadata:
+  name: keda-http-add-on-interceptor
+  namespace: keda
+  labels:
+    control-plane: interceptor
+    app: keda-http-add-on
+    app.kubernetes.io/name: keda-http-add-on
+    app.kubernetes.io/version: latest
+    app.kubernetes.io/component: interceptor
+    app.kubernetes.io/part-of: keda-http-add-on
+spec:
+  replicas: 1
+  selector:
+    matchLabels:
```

Comment:
I think we should add `app: keda-http-add-on` to all deployments (or `app.kubernetes.io/name`)

Reply:
you are right, omw